src/colvarcomp_distances.cpp

```diff
@@ -384,32 +384,30 @@ void colvar::distance_dir::apply_force(colvarvalue const &force)
   cvm::real const iprod = force.rvector_value * x.rvector_value;
   cvm::rvector const force_tang = force.rvector_value - iprod * x.rvector_value;
 
-  if (!group1->noforce)
-    group1->apply_force(-1.0 * force_tang);
-
-  if (!group2->noforce)
-    group2->apply_force(       force_tang);
+  if (!group1->noforce) {
+    group1->apply_force(-1.0 / dist_v.norm() * force_tang);
+  }
+  if (!group2->noforce) {
+    group2->apply_force( 1.0 / dist_v.norm() * force_tang);
+  }
 }
 
 
-cvm::real colvar::distance_dir::dist2(colvarvalue const &x1,
-                                      colvarvalue const &x2) const
+cvm::real colvar::distance_dir::dist2(colvarvalue const &x1, colvarvalue const &x2) const
 {
-  return (x1.rvector_value - x2.rvector_value).norm2();
+  return x1.dist2(x2);
```

Comment:
So now the square of distance of two `distanceDir` is calculated as the angle between them. Should we clarify that in the documentation?

After merging this PR and #713 , I may still need to verify if `polarPhi` and `distanceDir` with a custom function could yield the same trajectories.

Reply:
> So now the square of distance of two `distanceDir` is calculated as the angle between them. Should we clarify that in the documentation?

Yes. I meant to do that, but forgot. I would say, though, that defining the distance as such is the same as taking the standard Euclidean distance (L^2 norm) but making it comply with the unit-vector constraint.

> After merging this PR and #713 , I may still need to verify if `polarPhi` and `distanceDir` with a custom function could yield the same trajectories.

Yeah, that makes sense.